dist/init/systemd/rkt-api-tcp.socket

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=rkt api service socket
+PartOf=rkt-api.service
+
+[Socket]
+ListenStream=15441
```

Comment:
It can not work due lack of support for socket activation in api service.
There are 2 modes - one require knowledge about socket activation and so reusing already passed opened fd as listening socket (no for which is passed by systemd as environment variable). Second one - works per connection in old style inetd fashion - not supporting networking at all, only using stdin/stdout as already prepared communication channel.

Adding support for first one should be trivial - if there is in env specific variable, interpret it as integer value, use file descriptor with this number as base for constructing tcpl, skip manually setting tcpl with net.Listen(...).